tf_files/aws/modules/sqs/cloud.tf

```diff
@@ -0,0 +1,45 @@
+resource "aws_sqs_queue" "my_queue" {
+  name = var.sqs_name
+  # 5 min visilibity timeout; avoid consuming the same message twice
+  visibility_timeout_seconds = 300
+  # 1209600s = 14 days (max value); time AWS will keep unread messages in the queue
+  message_retention_seconds = 1209600
```

Comment:
Do we have any alerting/monitoring on the queues? Even with 14 days retention we could potentially lose messages in the queue. If that happened on a Commons running audit-service and we didn't detect it we could lose audit history. If there was an AWS outage/processing backup we could also potentially have a huge backlog for the audit-service to process.

Reply:
I haven't looked into that. We might be able to set up CloudWatch monitoring

Reply:
i created ticket DEVOPS-52 to look into this